vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/Main.java

```diff
@@ -2050,20 +2096,25 @@ private void load(HttpServerExchange exchange, UnloadedData unloadedData) {
                       : output.getUrl())
                   .getBytes(StandardCharsets.UTF_8));
           final var correctOutputId = hexDigits(fileDigest.digest());
-          if (!output.getId().equals(correctOutputId)) {
-            badRequestResponse(
-                exchange,
-                String.format(
-                    "Workflow run %s has output %s that should have ID %s.",
-                    workflowRun.getId(), output.getId(), correctOutputId));
-            return;
+
+          if (verify) {
+            if (!output.getId().equals(correctOutputId)) {
+              badRequestResponse(
+                  exchange,
+                  String.format(
+                      "Workflow run %s has output %s that should have ID %s.",
```

Comment:
`output`-> `analysis`? As the item at issue is in the `analysis` array, and `output` doesn't appear anywhere in the UnloadedData